### PR TITLE
Add podcaster group

### PR DESCRIPTION
**Pull request by Raemon**
Allows 'admins' and a new 'podcaster' user permissions group to edit the podcastEpisodeId field on posts.

Note: you can currently edit it to be a string that doesn't match an existing podcastEpisodeId – the result will just be that the post won't load any audio.

Also note: this doesn't currently enable podcasters to edit posts that they don't own. That requires somewhat-more-complex permissions updates for how users edit posts and gain access to the post-edit UI. I'm holding off on that until after Great Divergence Repairs, and/or until a future PR where we actually make a good podcastEpisode editing form control UI (when this code actually starts to matter more)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1202937105610037) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by b0b3rt — APPROVED** [review]

File: packages/lesswrong/lib/collections/posts/schema.tsx [hunk 1]
```diff
@@ -978,6 +978,9 @@ const schema: SchemaType<DbPost> = {
     }),
     optional: true,
     viewableBy: ['guests'],
+    insertableBy: ['admins', 'podcasters'],
+    editableBy: ['admins', 'podcasters'],
+    group: formGroups.audio,
```
Comment (oetherington) [inline packages/lesswrong/lib/collections/posts/schema.tsx:983]: The code changes look good to me, but the form field might need to be forum gated - I'm not sure if we're adding audio on the EA Forum. Maybe @jpaddison3 can advise?

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/posts/schema.tsx:983]: We sometimes Forum gate the ability to add fields the other Forum would not care about, and sometimes we don't. I mildly would prefer it, but not strongly.

Comment (oetherington) [reply packages/lesswrong/lib/collections/posts/schema.tsx:983]: The issue is that this adds an "Audio" option when creating a new post:
Screenshot 2022-09-09 at 17 09 44

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/posts/schema.tsx:983]: Is it auto-uncollapsed? Cause that would be annoying for anybody. Yeah, that upgrades me to medium caring

Comment (Raemon) [reply packages/lesswrong/lib/collections/posts/schema.tsx:983]: It should only appear for admins and people you've manually added to the podcast group. (Although I agree it should probably be collapsed by default)

Comment (jpaddison3) [reply packages/lesswrong/lib/collections/posts/schema.tsx:983]: I think even for admins, I don't like the extra category sitting unused on the EA Forum, and medium-care about it being Forum-gated. It's easy to do, I think.
```suggestion
    group: formGroups.audio,
    hidden: isEAForum,
```

**Pull request merged**